modules/cnf-topology-aware-lifecycle-manager-backup-feature.adoc

```diff
@@ -35,12 +35,13 @@ nodes:
     role: "master"
     rootDeviceHints:
         hctl: "0:2:0:0"
-        deviceName: /dev/sda
+        deviceName: /dev/disk/by-id/scsi-3600508b400105e210000900000490000
 ........
 ........
-    #Disk /dev/sda: 893.3 GiB, 959119884288 bytes, 1873281024 sectors
+    #Disk /dev/disk/by-id/scsi-3600508b400105e210000900000490000:
+    #893.3 GiB, 959119884288 bytes, 1873281024 sectors
     diskPartition:
-        - device: /dev/sda
+        - device: /dev/disk/by-id/scsi-3600508b400105e210000900000490000
```

Comment:
Could these actual serial numbers also be generalized to `<serial_number>` here, or is it intended to really show more of a true example?

Also, a bit out-of-scope for the PR, but in this block I am wondering about:

* The two `........` lines; should this be `# ...` for YAML per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting)?
* `hostName:` references what I believe is typically intended to be an internal-only domain; can this be generalized to more of an example server?

Preview: https://63453--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-talm-for-cluster-upgrades.html#talo-backup-start_and_update_cnf-topology-aware-lifecycle-manager

Reply:
> Could these actual serial numbers also be generalized to <serial_number> here, or is it intended to really show more of a true example?

the latter, just because of the other specific information associated with the disk

> The two ........ lines; should this be # ... for YAML per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting)?

probably! i'm happy to update this in this PR

> hostName: references what I believe is typically intended to be an internal-only domain; can this be generalized to more of an example server?

i'm not sure about this one, and would need to consult whoever the SME is for `cnf` in a followup PR. thought it's outside my area, I can submit this as a doc bug in JIRA 🙂

Reply:
updated to YAML ellipses and i've submitted the JIRA ticket, [OCPBUGS-18605](https://issues.redhat.com/browse/OCPBUGS-18605)